### PR TITLE
Accept source agreements when executing 'winget list'

### PR DESCRIPTION
**Pull request by ValdisThomann**
This allows the tool to work if the user has never run winget list before.

### CODE REVIEW

File: src/WinDebloat/WinGet.cs [hunk 1]
```diff
@@ -63,14 +63,25 @@ public static async Task<List<Package>> List()
             if (line.Length > 35)
             {
                 list.Add(new(
-                    line[..35].TrimEnd(),
```
Comment (SimonCropp) [inline src/WinDebloat/WinGet.cs:66]: why was this required?

Comment (ValdisThomann) [reply src/WinDebloat/WinGet.cs:66]: Without it, the id value for in the list had leading spaces, i.e. `          github.cli`

Comment (SimonCropp) [reply src/WinDebloat/WinGet.cs:66]: ahh ok. thanks
